src/convertToObject.js

```diff
@@ -1,12 +1,40 @@
 'use strict';
 
+/**
+ * @param {string} sourceString
+ *
+ * @return {object}
+ */
+'use strict';
```

Comment:
The 'use strict' directive is written twice in this file. The second occurrence on line 8 is redundant and should be removed to avoid confusion.

Reply:
Remove duplicates